src/MongoDB.Bson/MongoDB.Bson.csproj

```diff
@@ -14,7 +14,7 @@
     <Company>MongoDB Inc.</Company>
     <Copyright>Copyright © 2010-present MongoDB Inc.</Copyright>
     <Description>Official MongoDB supported BSON library. See http://www.mongodb.org/display/DOCS/CSharp+Language+Center for more details.</Description>
-    <Authors>rstam;craiggwilson</Authors>
+    <Authors>vincentkam;DmitryLukyanov;rstam;craiggwilson</Authors>
```

Comment:
Is `DmitryLukyanov` a valid Nuget username?

Reply:
Does the order of the authors carry any meaning?

Reply:
>Is DmitryLukyanov a valid Nuget username?

It's my `github` name

Reply:
re: ordering: Alphabetically based off last name (I took the format from Github contributors page), although I now see that it's harder to tell since we're using usernames. 

Fwiw, the "Authors" names don't link to NuGet profiles, it's just plain text (and pretty soon our NuGet profiles will be empty anyways since we'll be changing "Owner" to MongoDB).  We could also use our full names instead of our NuGet/GitHub usernames, as some other packages do

Reply:
I assumed (erroneously?) that these had to be Nuget usernames. 

If not, let's use human readable names. Just confirm to be 100% sure.

Alphabetical is fine.

Reply:
Oops, turns out the other package authors using their full names [may not be following the csproj spec](https://docs.microsoft.com/en-us/dotnet/core/tools/csproj)? I'm changing it to use Nuget usernames (still sorted by last name).

Reply:
Thanks for researching that we were doing the correct thing with author names.